test/jasmine/tests/legend_scroll_test.js

```diff
@@ -100,19 +101,19 @@ describe('The legend', function() {
         function dragScroll(element, rightClick) {
             var scrollBox = getScrollBox();
             var scrollBar = getScrollBar();
+            var scrollBarBB = scrollBar.getBoundingClientRect();
             var legendHeight = getLegendHeight(gd);
             var scrollBoxYMax = gd._fullLayout.legend._height - legendHeight;
             var scrollBarYMax = legendHeight -
-                constants.scrollBarHeight -
+                scrollBarBB.height -
                 2 * constants.scrollBarMargin;
             var initialDataScroll = scrollBox.getAttribute('data-scroll');
             var dy = 50;
             var finalDataScroll = '' + Lib.constrain(initialDataScroll -
                 dy / scrollBarYMax * scrollBoxYMax,
                 -scrollBoxYMax, 0);
 
-            var scrollBarBB = scrollBar.getBoundingClientRect();
-            var y0 = scrollBarBB.top + scrollBarBB.height / 2;
+            var y0 = scrollBarBB.top + scrollBarBB.height / 5;
```

Comment:
notice that I moved the drag point away from the center of the `scrollBar`, thereby testing that it drags from where you grab it rather than snapping to the center.